source/quickstart/windows.rst

```diff
@@ -6,31 +6,115 @@ Quickstart for Windows
+
+      https://dl.min.io/server/minio/release/windows-amd64/minio.exe
+      
+   Do not attempt to run the executable.
+   You call the executable to launch the server.
+
+#. Launch the :mc:`minio server`
+
+   In PowerShell or the Command Prompt, navigate to the location of the executable or add the path of the ``minio.exe`` file to the system ``$PATH``.
+   
+   Use this command to start a local MinIO instance in the ``C:\minio`` folder.
+   If desired, you can replace ``C:\minio`` with another location to use for the MinIO instance.
+
+   .. code-block:: shell
+      :class: copyable
+
+      .\minio.exe server C:\minio
+
+   The :mc:`minio server` process prints its output to the system console, similar to the following:
+
+   .. code-block:: shell
+
+      API: http://192.0.2.10:9000  http://127.0.0.1:9000
+      RootUser: minioadmin
+      RootPass: minioadmin
+
+      Console: http://192.0.2.10:9001 http://127.0.0.1:9001
+      RootUser: minioadmin
+      RootPass: minioadmin
+
+      Command-line: https://docs.min.io/docs/minio-client-quickstart-guide
+         $ mc alias set myminio http://192.0.2.10:9000 minioadmin minioadmin
+
+      Documentation: https://docs.min.io
+
+      WARNING: Detected default credentials 'minioadmin:minioadmin', we recommend that you change these values with 'MINIO_ROOT_USER' and 'MINIO_ROOT_PASSWORD' environment variables.
+
+#. Connect to the Server
+
+   In your browser, go to http://127.0.0.1:9000.
+
+   Access the :ref:`minio-console` by going to a browser (such as Microsoft Edge) and enter one of the Console address specified in the :mc:`minio server` command's output.
+   For example, :guilabel:`Console: http://192.0.2.10:9001 http://127.0.0.1:9001` in the example output indicates two possible addresses to use for connecting to the Console.
+
+   If you go to port ``9000``, which is used for connecting to the API, you are automatically redirected to the Console port.
+
+   Log in to the Console with the ``RootUser`` and ``RootPass`` user credentials displayed in the output.
+   These default to ``minioadmin | minioadmin``.
+
+   .. image:: /images/minio-console-dashboard.png
+      :width: 600px
+      :alt: MinIO Console Dashboard displaying Monitoring Data
+      :align: center
+
+   You can use the MinIO Console for general administration tasks like Identity and Access Management, Metrics and Log Monitoring, or Server Configuration. 
+   Each MinIO server includes its own embedded MinIO Console.
+
+   From the console, you can create and manage buckets, add objects, and many other tasks.
+   For more information, see the :ref:`minio-console` documentation.
+
+#. `(Optional)` Install the MinIO Client
+
+   The :ref:`MinIO Client <minio-client>` allows you to work with your MinIO volume from the commandline.
+
+   Download the standalone MinIO server for Windows from the following link:
+
+   https://dl.min.io/client/mc/release/windows-amd64/mc.exe
+
+   Double click on the file to run it.
```

Comment:
Does double clicking work for mc? interesting.

Reply:
Yes, you actually have to install it to make the `mc` command available at the prompt.

Reply:
Interesting - I'd recommend we say that so the behavior on run is explicit. This is an unknown binary so we should say what happens when it's executed outside of the commandline.